docs/consume-packages/Package-Restore.md

```diff
@@ -1,52 +1,52 @@
+If a package isn't already installed, NuGet first attempts to retrieve it from the [cache](../consume-packages/managing-the-global-packages-and-cache-folders.md). If the package isn't in the cache, NuGet then attempts to download the package from all enabled sources, which appear in the  **Tools** > **Options** > **NuGet Package Manager** > **Package Sources** list in Visual Studio. During restore, NuGet ignores the order of package sources, using the package from whichever source is first to respond to requests. For more information, see [Configure NuGet behavior](Configuring-NuGet-Behavior.md). 
 
 > [!Note]
-> NuGet does not indicate a failure to restore a package until all the sources have been checked. At that time, NuGet reports a failure for only the last source in the list. The error implies that the package wasn't present on *any* of the other sources, even though errors are not shown for each of those sources individually.
+> NuGet doesn't indicate a failure to restore a package until all the sources have been checked. At that time, NuGet reports a failure for only the last source in the list. The error implies that the package wasn't present on *any* of the other sources, even though errors aren't shown for each of those sources individually.
 
-Package restore is triggered in the following ways:
+You can trigger Package Restore in the following ways:
 
-- **dotnet CLI**: use the [dotnet restore](/dotnet/core/tools/dotnet-restore?tabs=netcore2x) command, which restores packages listed in the project file (see [PackageReference](../consume-packages/package-references-in-project-files.md)). With .NET Core 2.0 and later, restore is done automatically with `dotnet build` and `dotnet run`.
+- **Dotnet command line**: Use the [dotnet restore](/dotnet/core/tools/dotnet-restore?tabs=netcore2x) command to restore packages listed in the project file. With .NET Core 2.0 and later, restore happens automatically with `dotnet build` and `dotnet run`. For more information, see [PackageReference](../consume-packages/package-references-in-project-files.md). 
 
-- **Package Manager UI (Visual Studio on Windows)**: Packages are restored automatically when creating a project from a template and when building a project (subject to the option described in [Enabling and disabling package restore](#enabling-and-disabling-package-restore)). In NuGet 4.0+, restore also happens automatically when changes are made to a .NET Core SDK-based project.
+- **Package Manager** in Visual Studio for Windows: Package Restore happens automatically when you create a project from a template or build a project, subject to the options in [Enable and disable package restore](#enable-and-disable-package-restore)). In NuGet 4.0+, restore also happens automatically when you make changes to a .NET Core SDK-based project.
```

Comment:
I believe the term we've been using, only when necessary, is Visual Studio on Windows, which sounds less like a brand name than Visual Studio for Windows. If you have heard something different, let me know.